app/views/mechanics/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1><%= @mechanic.name %>'s Show Page</h1>
+
+<p>Name: <%= @mechanic.name %></p>
+<p>Years Experience: <%= @mechanic.years_experience %></p>
+
+<p><h3>Rides Worked On:</h3></p>
+<ol>
+  <% @rides.each do |ride| %>
+    <li><%= ride.name %></li>
+  <% end %>
+</ol>
+
+<p><h3>Add Ride to this Mechanic:</h3></p>
+<%= form_with url: mechanic_path(@mechanic), method: :patch, local: true, data: { turbo: false }  do |f| %>
+  <%= f.label :ride_id, "Ride ID:" %>
+  <%= f.text_field :ride_id %>
+  <%= f.submit 'Submit' %>
+<% end %>
```

Comment:
Everything is wonderful in this page but your Verb and Path for the form aren't exactly correct. We are CREATING a new relationship between a ride and a mechanic so this should be a `POST` instead of a `PATCH` and instead of the path being `"/mechanics/:id"`, it should be `"/mechanics/:id/rides"`.

Here is what the updated form should look like:

```erb
<%= form_with url: mechanic_rides_path(@mechanic), method: :post, local: true, data: { turbo: false }  do |f| %>
  <%= f.label :ride_id, "Ride ID:" %>
  <%= f.text_field :ride_id %>
  <%= f.submit 'Submit' %>
<% end %>
```